### ISSUE TITLE
How do you do semantic segmentation visualisation?

### ISSUE THREAD

**Issue by 2311762665** [ShiQiu0419/BAAF-Net#17]
How did you do the semantic segmentation visualization? Can you share the source code or introduce a tutorial on how to use it?

**Comment by ShiQiu0419** [comment]
We save the output and input data as numpy arrays, and then visualize them using open3d-python.

**Comment by 2311762665** [comment]
Yes, thanks for your answer, I found this part in the open source code you gave. But I am using it to generate discrete 3D scenes, how can I achieve the effect of 2D image rendering like in the paper?

**Comment by hhhhsmx** [comment]
> Yes, thanks for your answer, I found this part in the open source code you gave. But I am using it to generate discrete 3D scenes, how can I achieve the effect of 2D image rendering like in the paper?

Can you tell me which part of the visualization file it is? I didn't find it.thank you

**Comment by ShiQiu0419** [comment]
We use open3d library for visualization, please refer to its documentation for relevant functions. Thanks.


------------------ Original ------------------
From: hhhhsmx ***@***.***&gt;
Date: Thu,Oct 6,2022 5:03 PM
To: ShiQiu0419/BAAF-Net ***@***.***&gt;
Cc: ShiQiu0419 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [ShiQiu0419/BAAF-Net] How do you do semantic segmentation visualisation? (Issue #17)





  
Yes, thanks for your answer, I found this part in the open source code you gave. But I am using it to generate discrete 3D scenes, how can I achieve the effect of 2D image rendering like in the paper?
  
Can you tell me which part of the visualization file it is? I didn't find it.thank you
